torch/fx/experimental/symbolic_shapes.py

```diff
@@ -791,29 +891,71 @@ def _make_node_sizes_strides(method, func):
     def sizes_strides_impl(self, sizes, strides):
         op = getattr(sys.modules[__name__], method)
         if SYM_FUNCTION_MODE:
-            r = _handle_sym_dispatch(op, ([wrap_node(s) for s in sizes], [wrap_node(s) for s in strides]), {})
-            assert isinstance(r, SymBool), type(r)
-            return r.node
+            return to_node(
```

Comment:
We don't assert this is a bool anymore?

Reply:
Intentional, this invariant was relaxed in https://github.com/pytorch/pytorch/pull/94400 and now I'm applying it to the bool case now too.